modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -125,7 +124,7 @@
 	alt_titles = list("Roboticist", "Biomechanical Engineer", "Mechatronic Engineer")
 
 /datum/job/science_guard
-
+	alt_titles = list("Science Guard", "Science Officer", "Science Watchman")
```

Comment:
and again yep no "officer"

Reply:
no alt titles for this one pls

Reply:
For what reason?

Reply:
the only alt-title I'd want for any of the guard roles is "[Department] Guard"